index.html

```diff
@@ -1,12 +1,173 @@
+          <div class="icon icon3 no-hover">
+            <img class="no-hover" src="assets/images/music_icon.svg" alt="music box icon">
+          </div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Entertainment</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">Entertainment is core factor of the Back To Roots Global Concert, and be ready to be
+              blown away.</p>
+
+          </div>
+        </div>
+        <div class="box box4 container-fluid">
+          <div class="icon icon4 no-hover"><img class="no-hover" src="assets/images/donation_icon.svg"
+              alt="dollar coin icon"></div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Pledges <br /> & <br /> Donations</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">Most important part of the Back To Roots Global Concert is donations that will help
+              eliminate some of
+              global challenges through charities.</p>
+          </div>
+        </div>
+      </div>
+      <p class="no-display">SEE THE WHOLE PROGRAM</p>
+      <div class="btn-div card-box"><input type="button" value="Join BTR Global Concert 2012" class="join-btn"> </div>
+    </div>
+  </section>
+
+  <!-- guest artists -->
+  <section class="artists"></section>
+
+  <section class="about" id="about-page">
+    <div class="page-about">
+      <div class="overlay">
+        <div class="home-text about-text">
+          <h3>"It's about time to connect with your roots."</h3><br />
+          <h1>BACK TO ROOTS<br /> GLOBAL CONCERT 2012</h1>
+        </div>
+        <div class="home-p">
+          <p>A joyful celebration believing in the value of connecting with one's roots through musical entertainment.
+            Become the part of this year's concert by buying your ticket online.</p>
+        </div>
+        <div class="about-p2">
+          <p>Please contact us per Email for any further questions about the BTR Global Concert 2012!</p>
+          <p>info@brtgc.org</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <footer class="foot">
+    <p class="hide spon-h">SPONSORS</p>
+    <hr>
+    <div class="sponsor hide">
+      <img id="sp1" class="img-fluid" src="assets/images/moster.png" alt="monster energy drink logo">
+      <img id="sp2" class="img-fluid" src="assets/images/ford-logo.png" alt="ford logo">
+      <img id="sp3" class="img-fluid" src="assets/images/coca.png" alt="coca-cola logo">
+      <img id="sp4" class="img-fluid" src="assets/images/yamaha.png" alt="yamaha logo">
+    </div>
+    <p class="foot-p">2012 Back To Roots. Some rights Reserved.</p>
+  </footer>
+  <script src="main.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
 </body>
+
```

Comment:
## Please I repeat: You have to create **About** and **Home** separately in Both versions: **Desktop** and **Mobile**